test/unit/spec/OktaSignIn_spec.js

```diff
@@ -1,13 +1,31 @@
 /* eslint max-params:[0, 2], max-len:[2, 180] */
 define([
   'widget/OktaSignIn',
-  'helpers/util/Expect'
+  'helpers/util/Expect',
+  'util/Logger'
 ],
-function (Widget, Expect) {
-
+function (Widget, Expect, Logger) {
+  var signIn;
   var url = 'https://foo.com';
-  var signIn = new Widget({
-    baseUrl: url
+
+  beforeEach(function(){
+    spyOn(Logger, 'warn');
+    signIn = new Widget({
+      baseUrl: url
+    });
+  });
+
+  Expect.describe('Debug Mode', function () {
+    it('logs a warning message on page load', function () {
+      var debugMessage =
+      `
+        The Okta Sign-In Widget is running in development mode.
+        When you are ready to publish your app, embed the minified version to turn on production mode.
+        See: https://developer.okta.com/code/javascript/okta_sign-in_widget#cdn
+      `;
```

Comment:
Can we move these strings into our i18n library?

Reply:
I don't think we should spend time translating our debug messages. It would be a lot of extra work. If you feel really strongly about this @robertjd let me know.

Reply:
Ah, so I wasn't necessarily suggesting that we need to translate these.  I was just looking for a place where we could enumerate the strings so that we aren't repeating them.

Reply:
Ah, I see. I'd advocate for them staying here, since it's really only in the code base once (not counting tests). But I will defer to @jmelberg-okta 🙂